authorization-service/basicauthorizer.js

```diff
@@ -0,0 +1,54 @@
+"use strict";
+const dotenv = require("dotenv");
+dotenv.config();
+
+module.exports.basicAuthorizer = async (event, context, cb) => {
+  console.log(`Event: ${JSON.stringify(event)}`);
+
+  if (event.type !== "TOKEN") {
+    cb("Unauthorized");
+  }
+
+  try {
+    const authorizationToken = event.authorizationToken;
```

Comment:
I suggest to add next check
```
if (!authorizationToken) cb('Unauthorized');
```

Reply:
Good suggestion, thanks! will implement it during next task!